redaxo/src/addons/phpmailer/package.yml

```
@@ -10,6 +10,7 @@ page:
     icon: rex-icon rex-icon-envelope
     subpages:
         config: { title: 'translate:configuration' }
+        logging: { title: 'translate:logging' }
```

```suggestion
        log: { title: 'translate:logging' }
```

So heißt die Page unter System und im Cronjob-Addon auch, dann haben wir es einheitlich.
(Es muss dann aber auch die page-datei umbenannt werden)